src/ta/common.rs

```diff
@@ -599,7 +604,17 @@ impl fmt::Display for TrustAnchorSignerRequest {
                 }
             }
             writeln!(f)?;
-        }
+        }   
+        writeln!(f, "Certificates will be reissued {} weeks before expiry.", 
+            self.timing.issued_certificate_reissue_weeks_before)?;
+        if let Some(rt) = self.renew_time { 
+            writeln!(f, "The current certificate expires on {}.", rt.to_rfc3339())?; 
+            if let Some(weeks) = TimeDelta::try_weeks(self.timing.issued_certificate_reissue_weeks_before) {
+                let t = rt - weeks;
+                writeln!(f, "Ergo the certificate is eligible for renewal on {}.", t.to_rfc3339())?; 
```

Comment:
I would drop the word "Ergo", I would be surprised if it is commonly known amongst non-native English speakers, even among native English speakers.